apps/contracts/contracts/Zengo.sol

```diff
@@ -308,26 +305,19 @@ contract ZengoDAO is Constants, ZengoStorage, GStates, PermissionsEnumerable, Co
     //  TODO: Add function to add Evidence to Voting Iteration
     function addEvidence(
         uint256 _proposalId,
+        uint256 _timestamp,
```

Comment:
improve naming - same as the other comment.

Here the Context is different.

- The proposer already presented the proposal to the goverment department choosen and the timestamp will be the date a document was presented, or a replied was received.